dev/customer_testing/ci.dart

```diff
@@ -0,0 +1,80 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:path/path.dart' as p;
+
+/// To run this script, either:
+///
+/// ```sh
+/// cd dev/customer_testing
+/// dart ci.dart [sha]
+/// ```
+///
+/// Or:
+///
+/// ```sh
+/// ./dev/customer_testing/ci.sh [sha]
```

Comment:
Shouldn't this example be just:
```suggestion
/// ./dev/customer_testing/ci.sh
```
as it actually does not take (or forward to ci.dart) any parameters?